Dockerfile.erb

```diff
@@ -1,84 +1,38 @@
-FROM alpine:3.8
+FROM fluent/fluentd:v1.12.1-1.0
 
-ENV JDK_VERSION openjdk7
-RUN apk update && apk add --no-cache bash curl "${JDK_VERSION}-jre-base" ruby java-cacerts
+USER root
 
 # Add extra certificates. Specifically, rapidssl is needed because it's used
 # by Papertrail.
 ENV LOCAL_TRUST_DIR /usr/local/share/ca-certificates
-ADD rapidssl.crt "$LOCAL_TRUST_DIR/rapidssl.crt"
+COPY rapidssl.crt "$LOCAL_TRUST_DIR/rapidssl.crt"
```

Comment:
This is part of what I was talking about in [this Slack message](https://aptible-support.slack.com/archives/C022BUSK487/p1622814695034700?thread_ts=1622811611.031600&cid=C022BUSK487), but this probably isn't necessary anymore?
I'm assuming it's included in the papertrail plugin?

Reply:
ah, I see what you mean! It is most likely included in the papertrail plugin, but I won't feel comfortable removing it without testing first